js/map_utils.js

```diff
@@ -146,27 +115,33 @@ function updateView(newView, fireAnalytics = true) {
       nationDims.width/stateDims.width]);
   }
 
-  // set the styling: all states inactive for view=USA, just one state active
-  // otherwise. i tried doing this with .classed('active') and
-  // .classed('hidden') and css (conditional on activeView=='USA' and
-  // d.properties.STATE_ABBV === activeView), but that didn't work with transitions.
-  var states = map.selectAll('.state');
-  if(activeView === 'USA') {
-    hideCountyLines();
-    states
-      .transition()
-      .duration(750)
-      .style("fill", function(d) { return formatState('fill', d, false); })
-      .style("stroke", function(d) { return formatState('stroke', d, false); })
-      .style("stroke-width", function(d) { return formatState('stroke-width', d, false); });
-  } else {
-    showCountyLines(activeView);
-    states
+  // set the styling: setting by adding or removing class, so d3 transitions not used
```

Comment:
is this an aesthetic or performance decision? i agree that the aesthetics of doing it with classes are fine, but i don't necessarily prefer them to the d3 option, so just wondering why you switched.

Reply:
I explained it in more detail here https://github.com/USGS-VIZLAB/water-use-15/issues/95#issuecomment-378701304, but some was performance and some was actually getting the styles to work correctly by having them here.